spk/tvheadend/src/service-setup.sh

```diff
@@ -42,6 +47,18 @@ service_postinst ()
     $BIN/busybox --install $BIN
     $BIN/delgroup "${USER}" "users" >> ${INST_LOG}
     $BIN/deluser "${USER}" >> ${INST_LOG}
+
+    # EPG Grabber (zap2epg) - Create a Python virtualenv
+    ${VIRTUALENV} --system-site-packages ${PYTHONENV} >> ${INST_LOG}
+
+    # EPG Grabber (zap2epg) - Install the wheels/requirements
+    ${SYNOPKG_PKGDEST}/env/bin/pip install \
+             --no-deps --no-index --no-input --upgrade \
+             --force-reinstall --find-links \
+             ${WHEELHOUSE} ${WHEELHOUSE}/*.whl >> ${INST_LOG}  2>&1
```

Comment:
`>> ${INST_LOG}` is not supported anymore and must be removed. All output (stdout and stderr) goes to the installer log.

```suggestion
    ${VIRTUALENV} --system-site-packages ${PYTHONENV}

    # EPG Grabber (zap2epg) - Install the wheels/requirements
    ${SYNOPKG_PKGDEST}/env/bin/pip install \
             --no-deps --no-index --no-input --upgrade \
             --force-reinstall --find-links \
             ${WHEELHOUSE} ${WHEELHOUSE}/*.whl
```